generate/templates/sync-gateway/Dockerfile.ubuntu.template

```diff
@@ -49,6 +49,7 @@ CMD ["/etc/sync_gateway/config.json"]
 USER sync_gateway
 WORKDIR /home/sync_gateway
 
+VOLUME /var/log/sync_gateway
```

Comment:
Are we really sure that we want this? That means that when anyone runs this docker image, whether they want it or not, the Docker daemon will create an anonymous volume and map it to this location. That feels pretty presumptuous on our part. In particular, this feels really bad if SGW is run in K8S, since these auto-volumes will always just be on whatever specific host the container happens to land on.

Looking at the Server side, Server writes in a variety of places under `/opt/couchbase/var`, and our documentation recommends users mount that location to a path on their filesystem if they want persistence. But it's up to the user to do what they want to do.

Reply:
I view this as somewhat self documenting as "you need to mount this path", and to me it looks like server does the same thing https://github.com/couchbase/docker/blob/9bf4f784bac49ece9c90ce36bd63b0e0882d5729/enterprise/couchbase-server/7.6.1/Dockerfile#L143

This is basically to avoid having reports about this not being a read-only image and indicating to the user that this is something that needs to be updated. I'd rather it writes to a volume that they don't know about, and only learn about via their disks filling up, indicating they need to put it somewhere.

This isn't a critical fix, just adds some self documentation.